.github/workflows/deploy_prod.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy production
+
+on:
+  release:
+    types: [ released ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      DOCKERHUB_ORG: gnosispm
+      DOCKERHUB_PROJECT: safe-client-gateway
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # checks out all branches and tags, necessary for versioning /about endpoint
+
+      - name: Dockerhub login
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Deploy tag
+        run: bash scripts/deploy_docker.sh ${GITHUB_REF##*/}
```

Comment:
One suggestion for this is to use `${{ github.event.release.tag_name }}` it represents the tagged version – `vX.Y.Z`

Can probably be done in a separate PR

Reply:
that would be definitely more readable :+1:

Reply:
I think this is only available on events triggered by the type `release` (not 100% sure)